backend/yugioh/models.py

```diff
@@ -0,0 +1,119 @@
+from django.db import models
+
+from card.models import Card
+
+
+class YugiohCard(models.Model):
+    type = models.CharField(
+        max_length=100,
+    )
+
+    frame_type = models.CharField(
+        max_length=100,
+    )
+
+    description = models.CharField(
+        max_length=1000,
+        blank=True,
+        null=True,
+    )
+
+    attack = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    defense = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    level = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    race = models.CharField(
+        max_length=100,
+    )
+
+    attribute = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    archetype = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    image = models.URLField(
+        blank=True,
+        null=True,
+    )
+
+    card = models.ForeignKey(
```

Comment:
The `YugiohCard` should have a 1:1 relationship with the `Card` model, as per the diagram. I advise you to make the `YugiohCard` model inherit the card model. After that, fix the `__str__` representation.

Reply:
All necessary changes have been made.